ckan/tests/cli/test_cli.py

```diff
@@ -74,18 +74,105 @@ def test_command_from_extension_is_available_when_all_requirements_satisfied(cli
     assert not result.exit_code
 
 
-def test_ckan_config_loader_parse_files():
+def test_ckan_config_loader_parse_file():
     """
-    CKANConfigLoader should parse both 'test.ini.tpl' and 'test-core.ini.tpl'
-    files since test.ini.ptl has a use = config:test-core.ini.tpl config.
+    CKANConfigLoader should parse and interpolate variables in
+    test-core.ini.tpl file both in DEFAULT and app:main section.
     """
-    filename = os.path.join(os.path.dirname(__file__), u'data/test.ini.tpl')
+    data_dir = os.path.dirname(__file__) + u'/data/'
```

Comment:
`data` is perhaps a confusing word here (and also ignored by our .gitignore file). What about `config` or `templates`?